docker-compose.yml

```diff
@@ -17,6 +17,21 @@ services:
       MYSQL_PASSWORD: starchart_password
       MYSQL_ROOT_PASSWORD: root_password
 
+  test_database:
+    image: mysql:8
+    container_name: test_database
+    ports:
+      # Only for dev to run tests in a database
+      - 3307:3306
+    volumes:
+      # Persist data to ./mysql-data-test. Remove if you want to clear the db.
```

Comment:
Since this is for testing, why don't you remove this `volumes` block altogether, and not have it persist.  After the container is brought down, the data would be gone.  This seems like what you want for testing.